WFD/PreviewItem.py

```diff
@@ -0,0 +1,256 @@
+            for claim in ref.source_notest:
+                txt += ':*{}\n'.format(PreviewItem.format_claim(claim))
+
+        return txt
+
+    @staticmethod
+    def format_claim(claim):
+        """Create a preview for a single pywikibot.Claim."""
+        special = False
+        itis = claim.getTarget()
+        if claim.getSnakType() != 'value':
+            special = True
+            itis = claim.getSnakType()
+
+        return '{prop}: {itis}'.format(
+            prop=PreviewItem.make_wikidata_template(claim.id),
+            itis=PreviewItem.format_itis(itis, special)
+        )
+
+    @staticmethod
+    def format_itis(itis, special=False):
+        """
+        Create a preview for the itis component of a Statement.
+
+        :param itis: a Statement or the itis component of a Statement or
+            Qualifier.
+        :param special: if it is a special type of value (i.e. novalue,
+            somevalue). Is set automatically if a Statement is passed to itis.
+        """
+        # handle the case where a Statement was passed
+        if isinstance(itis, WdS.Statement):
+            special = itis.special
+            itis = itis.itis
+
+        if isinstance(itis, pywikibot.ItemPage):
+            return PreviewItem.make_wikidata_template(itis)
+        elif special:
+            return PreviewItem.make_wikidata_template(itis, special=True)
+        elif isinstance(itis, pywikibot.WbQuantity):
+            unit = itis.get_unit_item() or ''
+            amount = itis.amount
+            if unit:
+                unit = PreviewItem.make_wikidata_template(unit)
+            return '{} {}'.format(amount, unit).strip()
+        elif isinstance(itis, pywikibot.WbTime):
+            return itis.toTimestr()
+        else:
+            return str(itis)
+
+    def format_labels(self):
+        """
+        Create a label(s) preview.
+
+        The first label is italicised (to indicate it is the preferred label
+        whereas the rest are aliases).
+
+        :return: string
+        """
+        preview = ''
+        for lang, labels in self.labels_dict.items():
+            if labels:
+                labels = [PreviewItem.make_text_italics(labels[0])] + \
+                    labels[1:]
+                preview += '* {}: {}\n'.format(
+                    PreviewItem.make_text_bold(lang), ' | '.join(labels))
+        return preview
+
+    def format_descriptions(self):
+        """
+        Create a descriptions(s) preview.
+
+        :return: string
+        """
+        preview = ''
+        for lang, description in self.desc_dict.items():
+            preview += '* {}: {}\n'.format(
+                PreviewItem.make_text_bold(lang), description)
+        return preview
+
+    def format_item(self):
+        """
+        Create an item preview.
+
+        :return: string
+        """
+        if self.item:
+            return PreviewItem.make_wikidata_template(self.item)
+        else:
+            return 'New item created'
```

Comment:
You might be right but I'm loath to leave it completely blank.Changed it to `–`. Would that work?